### PR TITLE
chore: set up towncrier for changelog management

### PR DESCRIPTION
**Pull request by SdgJlbl**
## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: changes/861.changed [hunk 1]
```diff
@@ -0,0 +1 @@
+- [chore] `towncrier` is now used for changelog management ([#861](https://github.com/Substra/substra-backend/pull/861))
```
Comment (ThibaultFy) [inline changes/861.changed:1]: Do we need to specify "chore" in the message ?

Comment (ThibaultFy) [reply changes/861.changed:1]: (asking for my future self)

**Review by ThibaultFy — APPROVED** [review]

File: changes/861.changed (hunk 1 above)
Comment (SdgJlbl) [reply changes/861.changed:1]: no, the only thing that matters is the name of the file (has to end with `.added`, `.fixed`, `.removed` or `.changed`

you can put whatever you want inside the file, including nested multiline lists, code snippets...

Comment (ThibaultFy) [reply changes/861.changed:1]: okay I see thx

**Pull request merged**